xml/art_migration.xml

```diff
@@ -411,22 +435,24 @@
       <itemizedlist>
        <listitem>
         <para>
-         Ensure that the new deployer is able to reach ILO interfaces
+         Ensure that the seed node is able to reach ILO interfaces
          referenced in the <filename>servers.yml</filename> file:
         </para>
 <screen>(seed)> ping <replaceable>ILO_IP</replaceable></screen>
        </listitem>
        <listitem>
         <para>
-         If network accessibility to the ILOs requires special network
-         configuration and you would like to use &lcm; for that process based on
-         your model:
+         If you are not able to ping ILO IP addresses because network accessibility
```

Comment:
Isn't it `iLO` (it is at least according to [wikipedia](https://en.wikipedia.org/wiki/HP_Integrated_Lights-Out))?